### ISSUE TITLE
Not possible to disable, delete etc a user via a screen reader

### ISSUE THREAD

**Issue by caturria** [nextcloud/server#19464]
### Steps to reproduce
1. Navigate to users list.
2. Use arrow keys (or Control Option Right/ Control Option Left if on Mac OS) to move screen reader cursor to name of user.

### Expected behaviour
Popup menu should appear when name of user is interacted with via keyboard. Screen reader should indicate that the item will produce a popover menu.

### Actual behaviour
Nothing happens. The menu is triggered only by a mouseover event, which cannot be replicated by a screen reader.

## Server configuration detail

**Operating system:** Linux 3.10.0-1062.12.1.el7.x86_64 #1 SMP Tue Feb 4 23:02:59 UTC 2020 x86_64

**Webserver:** nginx/1.16.1 (fpm-fcgi)

**Database:** Mariadb 10.4.12

**PHP version:** 

7.3.14
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, cgi-fcgi, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, imap, intl, json, exif, mysqlnd, PDO, Phar, posix, shmop, SimpleXML, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlwriter, xsl, mcrypt, mysqli, pdo_mysql, pdo_sqlite, wddx, xmlreader, igbinary, imagick, memcache, msgpack, zip, memcached, redis, Zend OPcache

**Nextcloud version:** 17.0.3 - 17.0.3.1

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated from version as of early October 2019 (do not remember exact version number)

**Where did you install Nextcloud from:** Direct download from nextcloud.com

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.3.0
 - activity: 2.10.1
 - audioplayer: 2.9.0
 - calendar: 2.0.1
 - cloud_federation_api: 1.0.0
 - comments: 1.7.0
 - contacts: 3.1.8
 - dav: 1.13.0
 - federatedfilesharing: 1.7.0
 - federation: 1.7.0
 - files: 1.12.0
 - files_pdfviewer: 1.6.0
 - files_rightclick: 0.15.1
 - files_sharing: 1.9.0
 - files_trashbin: 1.7.0
 - files_versions: 1.10.0
 - files_videoplayer: 1.6.0
 - firstrunwizard: 2.6.0
 - gallery: 18.4.0
 - issuetemplate: 0.6.0
 - logreader: 2.2.0
 - lookup_server_connector: 1.5.0
 - mail: 1.1.2
 - nextcloud_announcements: 1.6.0
 - notifications: 2.5.0
 - oauth2: 1.5.0
 - password_policy: 1.7.0
 - phonetrack: 0.6.2
 - privacy: 1.1.0
 - provisioning_api: 1.7.0
 - ransomware_detection: 0.6.0
 - ransomware_protection: 1.5.1
 - recommendations: 0.5.0
 - serverinfo: 1.7.0
 - spreed: 7.0.2
 - support: 1.0.1
 - survey_client: 1.5.0
 - systemtags: 1.7.0
 - text: 1.1.1
 - theming: 1.8.0
 - twofactor_backupcodes: 1.6.0
 - updatenotification: 1.7.0
 - viewer: 1.2.0
 - workflowengine: 1.7.0
Disabled:
 - admin_audit
 - encryption
 - files_external
 - sharebymail
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "cloud.caturria.ca"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "17.0.3.1",
    "overwrite.cli.url": "https:\/\/cloud.caturria.ca",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "token_auth_enforced": true,
    "memcache.local": "\\OC\\Memcache\\Redis",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 6379,
        "timeout": 0,
        "password": "***REMOVED SENSITIVE VALUE***",
        "dbindex": 0
    },
    "installed": true,
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "maintenance": false,
    "mail_smtpmode": "smtp",
    "mail_smtpsecure": "tls",
    "mail_sendmailmode": "smtp",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauthtype": "PLAIN",
    "mail_smtpauth": 1,
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "587",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "theme": "",
    "loglevel": 2,
    "simpleSignUpLink.shown": false
}
```
</details>

**Are you using external storage, if yes which one:** 

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** 

## Client configuration

**Browser:** Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_2) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.4 Safari/605.1.15

**Operating system:** Mac OS Catalina

## Logs

<details><summary>Web server error log</summary>

```
Insert your web server log here 
```
</details>

<details><summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

<details><summary>Browser log</summary>

Insert your browser log here, this could for example include:

	a) The javascript console log
	b) The network log
	c) ...
</details>

**Comment by jancborchardt** [comment]
Thanks for the issue report @caturria and sorry for the issues! @GretaD something for you to look into?

**Comment by GretaD** [comment]
I have no touchscreen device to reproduce this, but can you @caturria please test the same thing for the file menu, or any other menu that is not in the user management page? Does it happen the same?

**Comment by caturria** [comment]
Hi,
Just to clarify, I do not use a touch screen device. I mostly use a Macintosh computer with a keyboard.
I have tested the actions menu in the files app, and it presents a fully accessible interface.
In Files, the ‘actions’ menu appears as a link to Voiceover. Pressing it reveals add to favourites, details, rename, move or copy, download and delete file immediately next to the actions link.
The user’s page, on the other hand, is problematic in a few ways. That actions menu is triggered exclusively by a mouse hover (and presumably disappears once the mouse is no longer over the user), which means that there is simply no means by which a keyboard user can interact with it.
Also, Mac specific: when scrolling through the users list using Control Option Right, the Voiceover cursor gets stuck in places.
Other platforms such as Windows present the former issue.
Kind regards,

Jordan.

**Comment by GretaD** [comment]
Thank you Jordan for all the details, i will have a look asap

**Comment by caturria** [comment]
Hi,
Thank you very much for taking these types of issues seriously and enabling minorities to benefit from Nextcloud as much as the majority can.
Kind regards,

Jordan.

**Comment by jancborchardt** [comment]
The fix is merged and will be in the next version, Nextcloud 19, which is set to release end of May.

Sorry for the issues @caturria, and your reporting and help is a huge support to us fixing the issues. If you find any other accessibility issue, please don’t hesitate to open further issues. Thank you so much!
